packages/data-stores/global.d.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * External dependencies
+ */
+declare module 'wpcom-proxy-request' {
+	import { WpcomRequestParams } from 'src/utils/wpcom-wrapper';
```

Comment:
Having the comment right above `declare module` doesn't seem quite right. I _think_ it's safe to move the import statement out of the block so it's at the top of the file.

Reply:
I'm actually wondering if `WpcomRequestParams` belongs in this file and that the utils should import it from here 🤔 
Seems like `wpcom-proxy-request` is the package that defines that object shape.

Reply:
> I think it's safe to move the import statement out of the block

Nope, you were right, just tried it :D